public/app/features/alerting/unified/components/rules/state-history/ErrorMessageRow.tsx

```diff
@@ -0,0 +1,38 @@
+import { css } from '@emotion/css';
+import { GrafanaTheme2 } from '@grafana/data';
+import { t } from '@grafana/i18n';
+import { Text, useStyles2 } from '@grafana/ui';
+
+interface ErrorMessageRowProps {
+  message: string;
+}
+
+export function ErrorMessageRow({ message }: ErrorMessageRowProps) {
+  const styles = useStyles2(getStyles);
+
+  return (
+    <div className={styles.errorRow} data-testid="state-history-error">
+      <div className={styles.errorContainer}>
+        <Text variant="bodySmall" truncate element="p">
+          <strong>{t('alerting.state-history.error-message-prefix', 'Error message:')}</strong> {message}
+        </Text>
+      </div>
+    </div>
+  );
+}
+
+const getStyles = (theme: GrafanaTheme2) => ({
+  errorRow: css({
+    color: theme.colors.text.secondary,
+    marginTop: theme.spacing(0.5),
+    display: 'block',
+  }),
+  errorContainer: css({
```

Comment:
You might consider using `Alert`, `Stack` and `Box` components from `grafana/ui` to avoid custom styling when they meet your needs